src/app/client/src/app/modules/dashboard/components/data-chart/data-chart.component.ts

```diff
@@ -109,7 +109,7 @@ export class DataChartComponent implements OnInit, OnDestroy {
           this.dateFilterReferenceName = filter.reference;
         }
         this.filtersFormGroup.addControl(_.get(filter, 'reference'), this.fb.control(''));
-        filter.options = _.uniq(_.map(this.chartData, data => data[filter.reference].toLowerCase()));
+        filter.options = _.sortBy(_.uniq(_.map(this.chartData, data => _.capitalize(data[filter.reference].toLowerCase()))));
```

Comment:
1 - _.sortBy will work fine. But in the case of alphanumeric cases where the initial word is common, it might cause some issue. For example:- Class 1, Class 2, Class 3, Class 4 ......

2- _.capitilize should be changed also in the subscription part. Since the matching is done with casing on. 
A better approach would be to use the angular pipe to capitalize on the initial word.

Reply:
1. Yeah works for initial word is common 
![Screenshot from 2020-04-08 10-32-45](https://user-images.githubusercontent.com/31840583/78746426-6ed02480-7984-11ea-8b7b-c93f6e44e6d0.png)